src/convertToObject.js

```diff
@@ -12,7 +12,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const keys = sourceString.split(';');
+  const newObj = {};
+
+  for (let i = 0; i < keys.length; i++) {
+    const arr = keys[i].split(':');
+
+    if (arr.length !== 2) {
+      continue;
+    }
+
+    const key = `${arr[0].trim()}`;
+    const value = `${arr[1].trim()}`;
```

Comment:
```suggestion
    const key = arr[0].trim();
    const value = arr[1].trim();
```
Навіщо тут інтерполяція?